app/web/src/newhotness/Explore.vue

```diff
@@ -499,14 +438,61 @@ const placeholderSearchText = computed(
   () => `Search across ${componentListRaw.data.value?.length ?? 0} Components`,
 );
 const componentList = computed(() => componentListRaw.data.value ?? []);
-const componentsById = computed(() =>
-  Object.fromEntries(componentList.value.map((c) => [c.id, c])),
-);
-
-const scrollRef = ref<HTMLDivElement>();
 
 const filteredComponents = reactive<ComponentInList[]>([]);
 
+// NOTE(nick,victor): this is also computed per-tile, likely due to the ongoing performance work.
```

Comment:
why not compute this _one time_ and put it on the EXPLORE CONTEXT that each tile can just re-use?

Reply:
That's a good idea! I followed the upgrade function logic. Can the computed set become out of date if used in the context?

Reply:
done